Sources/SwiftDocC/Semantics/Symbol/Symbol.swift

```diff
@@ -323,12 +332,75 @@ public final class Symbol: Semantic, Abstracted, Redirected, AutomaticTaskGroups
         self.redirectsVariants = redirectsVariants
         self.originVariants = originVariants
         self.automaticTaskGroupsVariants = automaticTaskGroupsVariants
-        self.extendedModule = extendedModule
     }
     
     public override func accept<V: SemanticVisitor>(_ visitor: inout V) -> V.Result {
         return visitor.visitSymbol(self)
     }
+
+    /**
+     Append a new generic constraint for the given extended module
+
+     - Parameters:
+        - extendedModule: Which module this symbol extends.
+        - typeKind: The ``SymbolGraph/Symbol/KindIdentifier`` of the symbol this extension extends.
+        - constraint: The new generic constraints to add
+        - trait: Which generic constraints variant to append to
+     - Returns: True if the constraint was appended to an existing or new swift extension variant. False if there was an existing swift extension variant for a different module.
```

Comment:
This function doesn't have a return value.

Reply:
Removed